en/Plugins/Getting started/Mobile development.md

```diff
@@ -18,6 +18,17 @@ To disable mobile emulation, enter the following and press `Enter`:
 this.app.emulateMobile(false);
 ```
 
+## Inspecting the webview on the actual mobile device
+
+### Android
+
+You can inspect Obsidian running on an Android device if you enable USB Debugging in Developer settings of Android. Then go to a chromium based browser on your desktop/laptop and navigate to chrome://inspect/. If you did everything right, if you have your phone/tablet connected to your PC via USB and the browser open at that link you should see your device pop up and it will let you run the usual devtools from there on it.
+
+### iOS
+
```

Comment:
This won't work for versions before iOS 16.4, and needs some setup on both ends first.
You can link to: https://webkit.org/web-inspector/enabling-web-inspector/ for more instructions